src/framework/preferences/css/stylus/utils/Themes.styl

```diff
@@ -1,250 +1,374 @@
-    }
-    ".fl-theme-bbr": {
-        foregroundColor: #000000,
-        backgroundColor: #bb9966,
-        linkColor: #000099,
-        disabledColor: #600000,
-        selectedForegroundColor: #ffffff,
-        selectedBackgroundColor: #000070,
-        buttonForegroundColor: #000000,
-        buttonBackgroundColor: #ffd18a
-    }
+.fl-theme-bw {
+    --fl-fgColor: #000000;
+    --fl-bgColor: #ffffff;
 }
 
-build-themes-prefsEditor(contrastThemes) {
+.fl-theme-wb {
+    --fl-fgColor: #ffffff;
+    --fl-bgColor: #000000;
+}
 
-    for themeName in contrastThemes {
-        fColor = contrastThemes[themeName].foregroundColor;
-        bColor = contrastThemes[themeName].backgroundColor;
+.fl-theme-yb {
+    --fl-fgColor: #ffff00;
+    --fl-bgColor: #000000;
+}
 
-        {themeName} {
-            .fl-prefsEditor {
+.fl-theme-by {
+    --fl-fgColor: #000000;
+    --fl-bgColor: #ffff00;
+}
 
-                // ON/OFF switch
-                .fl-switchUI-control:focus {
-                    box-shadow: 0 0 0 0.2em bColor,
-                                0 0 0 0.35em fColor;
-                }
+.fl-theme-lgdg {
+    --fl-fgColor: #bdbdbb;
+    --fl-bgColor: #555555;
+}
 
-                .fl-switchUI-controlKnob {
-                    background-color: fColor !important;
-                }
-            }
-        }
+.fl-theme-gd {
+    --fl-fgColor: #888888;
+    --fl-bgColor: #222222;
+    --fl-linkColor: #8080ff;
+    --fl-disabledColor: #cc0000;
+    --fl-selectedFgColor: #ffffff;
+    --fl-selectedBgColor: #008000;
+    --fl-buttonFgColor: #ffffff;
+    --fl-buttonBgColor: #000000;
+}
 
-        .fl-prefsEditor .fl-choice label {
-            &{themeName},
-            &{themeName} span,
-            &{themeName} .fl-crossout {
-                background-color: bColor !important;
-                border-color: fColor !important;
-                color: fColor !important;
-            }
-        }
-    }
+.fl-theme-gw {
+    --fl-fgColor: #6c6c6c;
+    --fl-bgColor: #ffffff;
+    --fl-linkColor: #6666cc;
+    --fl-disabledColor: #993333;
+    --fl-selectedFgColor: #ffffff;
+    --fl-selectedBgColor: #404080;
+    --fl-buttonFgColor: #000000;
+    --fl-buttonBgColor: #ffffff;
 }
 
-// For the separated panel
-get-colors-for-separatedPanelPrefsEditor(fColor, bColor) {
-    background-color: bColor;
-    border-color: fColor;
-    color: fColor;
+.fl-theme-bbr {
+    --fl-fgColor: #000000;
+    --fl-bgColor: #bb9966;
+    --fl-linkColor: #000099;
+    --fl-disabledColor: #600000;
+    --fl-selectedFgColor: #ffffff;
+    --fl-selectedBgColor: #000070;
+    --fl-buttonFgColor: #000000;
+    --fl-buttonBgColor: #ffd18a;
 }
 
-build-themes-separatedPanelPrefsEditor(contrastThemes) {
-    for themeName in contrastThemes {
-        fColor = contrastThemes[themeName].foregroundColor;
-        bColor = contrastThemes[themeName].backgroundColor;
-
-        {themeName} {
-            background-image: none !important;
```

Comment:
I believe this is still needed. If you look at the UIOptions demo the background image for the page is still in effect after the contrast theme is applied. See the [demo](https://build.fluidproject.org/infusion/demos/uiOptions/) from the build site as a comparison.

Reply:
@jobara Resolved in 4345c44b50466de62f05d47ccdb696365ea50c3c.